File: solutions/typescript/darts/2/darts.ts
export function score(x: number, y: number): number {
  const r = Math.sqrt(x * x + y * y);

  if (r > 5 && r <= 10) {
    return  1;
  }
  if (r > 1 && r <= 5) {
    return 5;
  }
  if (r <= 1) {
    return 10;
  }
  
  return 0;
}

File: solutions/python/darts/2/darts.py
def score(x, y):
    radius_squared = x ** 2 + y ** 2
    if radius_squared <= 1 :
        result_score = 10
    elif radius_squared <= 25:
        result_score = 5
    elif radius_squared <= 100:
        result_score = 1
    elif radius_squared > 100:
        result_score = 0

    return result_score
